src/shapes/Text/TextSVGExportMixin.ts

```diff
@@ -293,7 +293,7 @@ export class TextSVGExportMixin extends FabricObjectSVGExportMixin {
    * @return {String}
    */
   getSvgStyles(this: TextSVGExportMixin & Text, skipShadow?: boolean) {
-    // @ts-ignore this line is not error free. ( super is not the same type as this )
+    // @ts-expect-error TS doesn't respect this type casting
```

Comment:
to my vscode thise line wasn't an error and LINT was complaining i was using an unused expect-error. Don't you have that?

Reply:
here it is in current master 
<img width="658" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/0224fdc3-1ce7-4ce5-9c0f-8ef0e71057da">

Reply:
Same
But the rollup ts plugin throws